packages/core/contracts/chainbridge/SinkGovernor.sol

```diff
@@ -12,6 +12,10 @@ contract SinkGovernor {
 
     event ExecutedGovernanceTransaction(address indexed to, bytes indexed data);
 
+    /**
+     * @notice Constructor.
```

Comment:
Additionally, as noted in the pull request, none of the new functions follow the [Ethereum Natural Specification Format (NatSpec)](https://docs.soliditylang.org/en/latest/natspec-format.html) for their arguments. Consider including them.